test/test_corpus.cc

```diff
@@ -108,7 +108,8 @@ class CFGCollectorAndTyper {
             return m;
         }
         auto cfg = cfg::CFGBuilder::buildFor(ctx.withOwner(m->symbol), *m);
-        cfg = infer::Inference::run(ctx.withOwner(cfg->symbol), move(cfg));
+        auto symbol = cfg->symbol;
```

Comment:
clang-tidy informed me that the `cfg->symbol` and `move(cfg)` are not ordered, so this could become a use-after-move.

Reply:
They are ordered in c++17 that we use